src/widgets/NftDetailWidget/NftDetailWidget.scss

```diff
@@ -43,7 +40,65 @@
         width: calc(100% - var(--page-padding) * 2);
 
         @include for-size(tablet-landscape-up) {
+            position: relative;
+            width: 75%;
+            bottom: auto;
+        }
+    }
+
+    @include for-size(tablet-landscape-up) {
```

Comment:
This should be above all children. All children should have it's own media query. Look at other components like `WalletConnector.scss` how it's done.